css/v202209/symbiota/header.css

```diff
@@ -189,10 +208,53 @@ ul.menu {
   top: 0;
 }
 
+#lang-select-li {
+  margin-top: 1em;
+  text-align: center;
+}
+
 /* Media queries
 ========================================================================== */
 /* Medium screens and up */
 @media (min-width: 768px) {
+
+  .side-menu:not(:checked) ~ nav.top-menu .menu,
+  .side-menu:checked ~ nav.top-menu .menu {
+    display:block;
+  }
+
+  nav.top-menu > ul > li ul {
+    display: none;
+  }
+  
+  /* First level drop-down menu */
+  nav.top-menu > ul > li > a:hover > ul,
+  nav.top-menu > ul > li > a:focus > ul,
+  nav.top-menu > ul > li:hover > ul,
+  nav.top-menu > ul > li:focus-within > ul {
+    display: block;
+    position: absolute;
+    background-color: var(--menu-top-bg-color);
+    box-shadow: 3px 3px 3px #000;
+    width: max-content;
+  }
+
+  /* second+ level drop-down menu */
+  nav.top-menu > ul ul > li > a:hover > ul,
+  nav.top-menu > ul ul > li > a:focus > ul,
+  nav.top-menu > ul ul > li:hover > ul,
+  nav.top-menu > ul ul > li:focus-within > ul {
+    display: block;
+    position:absolute;
+    background-color: var(--menu-top-bg-color);
+    box-shadow: 3px 3px 3px #000;
+    width: max-content;
+  }
+  
+  /* Change this in order to change the Dropdown symbol */
+  nav.top-menu li > a:after { content:  ' +'; }
+  nav.top-menu li > a:only-child:after { content: ''; }
```

Comment:
I'm having trouble visualizing why this is necessary.

Reply:
Provides a visual clue as to which elements in the menu have sub menu items.